Experimental functionality cleanup (#5699)
mattermost/docs: source/onboard/shared-channels.rst

  :target: https://mattermost.com/deploy
  :alt: Available for Mattermost Self-Hosted deployments.

> Shared channels is an experimental feature that bring people together from multiple Mattermost installations, such as teams collaborating with external partners and customers using multiple Mattermost instances in a federated architecture.

Reviewer: ```suggestion
Shared channels is an experimental feature that brings people together from multiple Mattermost installations. For example, teams collaborating with external partners and customers using multiple Mattermost instances in a federated architecture.
```
